iroh-net/src/hp/netcheck/reportgen.rs

```diff
@@ -788,35 +788,29 @@ async fn run_probe(
                 }
             }
         }
-        Probe::Https { ref region, .. } => {
-            debug!(icmp=%pinger.is_some(), "sending probe HTTPS");
-
-            let res = if let Some(ref pinger) = pinger {
-                tokio::join!(
-                    time::timeout(
-                        ICMP_PROBE_TIMEOUT,
-                        measure_icmp_latency(region, pinger).map(Some)
-                    ),
-                    measure_https_latency(region)
+        Probe::Icmp { .. } => {
+            if let Some(ref pinger) = pinger {
+                match time::timeout(
+                    ICMP_PROBE_TIMEOUT,
+                    measure_icmp_latency(pinger, derp_node, derp_addr),
                 )
-            } else {
-                (Ok(None), measure_https_latency(region).await)
-            };
-            if let Ok(Some(icmp_res)) = res.0 {
-                match icmp_res {
-                    Ok(d) => {
-                        result.delay = Some(d);
+                .await
+                {
+                    Ok(Ok(latency)) => {
+                        result.delay = Some(latency);
                         result.ipv4_can_send = true;
                         result.icmpv4 = true;
                     }
-                    Err(err) => {
-                        warn!("icmp latency measurement failed: {:?}", err);
-                    }
+                    Ok(Err(err)) => warn!("icmp latency measurement failed: {:#}", err),
+                    Err(elapsed) => warn!("icmp latency measurement failed: {:#}", elapsed),
```

Comment:
maybe call this `measurement timed out`?

Reply:
47e8ba819e2b9f29a5601c4b913a94e4648e2331